pkg/apis/eventing/v1beta1/broker_validation.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	"knative.dev/pkg/apis"
+	"knative.dev/pkg/kmp"
+)
+
+const (
+	brokerClassAnnotationKey = "eventing.knative.dev/broker.class"
```

Comment:
We want to export `brokerClassAnnotationKey` so other impls can use it.

`brokerClassAnnotationKey ` -> `BrokerClassAnnotationKey`

Reply:
We also might want to add some helper methods to the broker type like 

```
func (b *Broker) IsMatchingClass(class string) bool {
  // TODO
}
```

exact naming TBD...